### PR TITLE
python311Packages.augmax: unstable-2022-02-19 -> 0.3.1

### PR DESCRIPTION
**Pull request by natsukium**
## Description of changes

Diff: https://github.com/khdlr/augmax/compare/3e5d85d6921a1e519987d33f226bc13f61e04d04...v0.3.1

Changelog: https://github.com/khdlr/augmax/blob/v0.3.1/CHANGELOG.md

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 261564` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.augmax</li>
    <li>python310Packages.augmax.dist</li>
    <li>python311Packages.augmax</li>
    <li>python311Packages.augmax.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/augmax/default.nix [hunk 1]
```diff
@@ -3,22 +3,29 @@
 , fetchFromGitHub
 , jax
 , jaxlib
+, pythonOlder
+, setuptools
```
Comment (samuela) [inline pkgs/development/python-modules/augmax/default.nix:7]: nit: i prefer to keep arguments a-z sorted in derivations i maintain

**Review by samuela — CHANGES_REQUESTED** [review]
thanks @natsukium ! just one nit

File: pkgs/development/python-modules/augmax/default.nix (hunk 1 above)
Comment (natsukium) [reply pkgs/development/python-modules/augmax/default.nix:8]: Yes, of course I know that, but I accidentally forgot to sort by alphabetical order.
Thanks.

**Review by samuela — APPROVED** [review]
diff LGTM thanks @natsukium !

**Pull request merged**